### ISSUE TITLE
[Playground] Checkbox, Btton -In dark theme the label for components are not visible.

### ISSUE THREAD

**Issue by sakshi173** [unoplatform/uno#11357]
### Current behavior

![Dark theme issue](https://user-images.githubusercontent.com/74563602/219540338-a574f778-0297-42a6-8064-dd9976067dd0.png)
![Dark theme issue 2](https://user-images.githubusercontent.com/74563602/219540353-0a987b8d-1272-459e-b23b-fc3ece784321.png)

https://user-images.githubusercontent.com/74563602/219540373-85707d55-2a6b-4d3f-bf69-bb977750365f.mp4

### Expected behavior

In dark theme components label should display correctly.

### How to reproduce it (as minimally and precisely as possible)

1. Visit https://playgroundcanary.z19.web.core.windows.net/#passwordbox.
2. Check 'Dark Theme' option.
3. Go to the Snippets Listview.
4. Click on 'Checkbox' or 'Button' snippets option.
5. Observe that label is not correctly displaying for components.

### Works on UWP/WinUI

None

### Affected platforms

WebAssembly

**Comment by Youssef1313** [comment]
Looks like a problem in Playground default styles?

https://github.com/unoplatform/Uno.Playground/blob/64c2ccfd3d10d8df0f68b7806f6ba17d6c4e79d7/src/Uno.Playground.Shared/DefaultStyles.xaml#L814

**Comment by Youssef1313** [comment]
@sakshi173 
![image](https://user-images.githubusercontent.com/31348972/221368316-4db828c4-df7c-4c8f-a375-938c30869808.png)

Does this look good to you?

**Comment by sakshi173** [comment]
@Youssef1313, still I am seeing Button style is not correctly visible for dark theme. below is the screenshot from the current canary build.
![button issue playground](https://user-images.githubusercontent.com/74563602/225191881-0fae4e25-44ce-4301-8883-59a37058c28b.png)

**Comment by Youssef1313** [comment]
@sakshi173 Yes, the PR https://github.com/unoplatform/Uno.Playground/pull/424 is not yet merged.
